src/query/v2/db_accessor.hpp

```diff
@@ -241,8 +246,12 @@ class DbAccessor final {
  public:
   explicit DbAccessor(storage::v3::Storage::Accessor *accessor) : accessor_(accessor) {}
 
-  std::optional<VertexAccessor> FindVertex(storage::v3::Gid gid, storage::v3::View view) {
-    auto maybe_vertex = accessor_->FindVertex(gid, view);
+  // TODO(jbajic) Fix Remove Gid
+  // NOLINTNEXTLINE(readability-convert-member-functions-to-static)
+  std::optional<VertexAccessor> FindVertex(uint64_t /*unused*/) { return std::nullopt; }
```

Comment:
Why did you keep this?

Reply:
Just for the sake of triggers, so I can minimize changes done there